pumpkin-solver/src/engine/conflict_analysis/conflict_analysis_context.rs

```diff
@@ -173,7 +173,7 @@ impl ConflictAnalysisContext<'_> {
                 let _ = self.internal_parameters.proof_log.log_inference(
                     self.propagator_store.get_tag(*propagator),
                     explanation_literals.iter().map(|&lit| !lit),
-                    None,
+                    self.assignments_propositional.false_literal,
```

Comment:
Is there a reason for the rest of these changes?

Reply:
Yes, the API of drcp-format-v0.1.0 does not take `Option<Literal>` but `Literal`